src/components/editMeasure/testCases/components/editTestCase/qiCore/EditTestCase.tsx

```diff
@@ -427,7 +429,7 @@ const EditTestCase = (props: EditTestCaseProps) => {
         }
         resetForm({ values: _.cloneDeep(nextTc) });
         handleHapiOutcome(nextTc?.hapiOperationOutcome);
-        if (["Pending", "Validating"].includes(tc.testCaseValidationStatus)) {
+        if (["Pending", "Validating"].includes(tc.validationStatus)) {
```

Comment:
Now we have enum. I think we can start using it wherever it is hardcoded?
```suggestion
        if ([ValidationStatus.Pending, ValidationStatus.Validating].includes(tc.validationStatus)) {
```

Reply:
@adongare I have changed it, thanks!